packages/help-center/src/hooks/use-still-need-help-url.tsx

```diff
@@ -1,26 +1,15 @@
 /* eslint-disable no-restricted-imports */
 
 import { useSupportAvailability } from '../data/use-support-availability';
-import { useIsWapuuEnabled } from './use-is-wapuu-enabled';
-
-/**
- * This function is used as a fallback for users navigating through wp-admin.
- * It checks if the 'flags' URL parameter is set to 'wapuu' when useIsWapuuEnabled() fails to do so.
- * @returns boolean
- */
-export function isWapuuFlagSetInURL(): boolean {
-	const currentUrl = window.location.href;
-	const urlParams = new URLSearchParams( new URL( currentUrl ).search );
-	return urlParams.get( 'flags' ) === 'wapuu';
-}
+import { useShouldUseWapuu } from './use-should-use-wapuu';
 
 export function useStillNeedHelpURL() {
 	const { data: supportAvailability, isLoading } = useSupportAvailability();
-	const isWapuuEnabled = useIsWapuuEnabled() || isWapuuFlagSetInURL();
+	const shouldUseWapuu = useShouldUseWapuu();
 	const isFreeUser = ! supportAvailability?.is_user_eligible;
 
 	if ( ! isFreeUser ) {
-		const url = isWapuuEnabled ? '/odie' : '/contact-options';
+		const url = shouldUseWapuu ? '/odie' : '/contact-options';
```

Comment:
Maybe we should change this to:

```js
if ( shouldUseWapuu ) {
	return { '/odie', isLoading: false };
} else {
	return { url: '/contact-form?mode=FORUM', isLoading };
}
```
Odie is available to all non-free users, so `'/contact-options'` will never be returned.